CMakeLists.txt

```diff
@@ -159,8 +159,10 @@ if(CMAKE_COMPILER_IS_GNUCC)
   endif()
 endif()
 
+# Set compiler optimizations for release
+set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_DEBUG} -O2")
```

Comment:
This should probably be:
set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -O2")
instead of
set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_DEBUG} -O2")